apps/meteor/app/custom-oauth/server/custom_oauth_server.js

```diff
@@ -362,6 +362,7 @@ export class CustomOAuth {
 				const update = {
 					$set: {
 						name: serviceData.name,
+						email: serviceData.email,
```

Comment:
`email` is not an attribute on IUser, the email addresses are stored in the `emails` list.
It should also only be changed if the keyfield is set to username.

```suggestion
						...(this.keyField === 'username' && serviceData.email && { emails: [{ address: serviceData.email, verified: true }] }),
```